### DISCUSSION TITLE
Difficulties using vue js & shopify plugins?

### DISCUSSION THREAD

**Discussion by soheimam** [uicrooks/shopify-theme-lab#68] [Q&A]
Hey team, first off great work! I really enjoyed this project and I can tell you have worked very hard on it, thank you. I just wanted to know if you have had any difficulty with this theme and customers adding additional Shopify apps to it? Would it even be possible for users to add additional apps into this theme?

**Comment by sergejcodes** [comment]
Hi 👋, 

users should be able to add Shopify apps/plugins. Hadn't had any difficulties with Shopify plugins so far. As long as there is [{{ content_for_header }}](https://github.com/uicrooks/shopify-theme-lab/blob/master/shopify/layout/theme.liquid#L24) tag in the header, everything should usually be fine.
